src/hotspot/share/classfile/classLoaderData.hpp

```diff
@@ -205,12 +205,14 @@ class ClassLoaderData : public CHeapObj<mtClass> {
   bool has_modified_oops()               { return _modified_oops; }
 
   oop holder_no_keepalive() const;
+  // Resolving the holder keeps this CLD alive for the current GC cycle.
   oop holder() const;
+  void keep_alive() const { (void)holder(); }
 
   void classes_do(void f(Klass* const));
 
  private:
-  bool keep_alive() const       { return _keep_alive > 0; }
+  bool keep_alive_ref_count() const { return _keep_alive_ref_count; }
```

Comment:
keep_alive_ref_count() should return an int. The Windows GHA build complains about this.

Reply:
I don't know why the linux compiler I used didn't, but fixed.